lib/rmt/mirror.rb

```diff
@@ -0,0 +1,86 @@
+module RMT
+end
+
+class RMT::Mirror
+
+  def initialize(mirroring_base_dir, repository_url, local_path, mirror_src = false, logger = nil)
```

Comment:
A method with so many arguments should use [keyword arguments](https://robots.thoughtbot.com/ruby-2-keyword-arguments). Then the ordering does not matter anymore and an invocation is way more readable ... I had to check the method definition here to understand what was going on in `bin/rmt-mirror`